packages/eas-cli/src/log.ts

```diff
@@ -48,6 +48,11 @@ function log(...args: any[]) {
   consoleLog(...args);
 }
 
+// for testability since default exports cannot be spied upon
```

Comment:
```
const mockLog = {
  __esModule: true, // this property makes it work
  default: jest.fn(toLog => toLog),
};
(mockLog.default as any).newLine = jest.fn();
jest.mock('../../../../log', () => mockLog);
```
I didn't test that, just copied  it from https://github.com/expo/eas-cli/pull/203/files